internal/cli/streams/instance/create_test.go

```diff
@@ -63,3 +110,12 @@ func TestCreateBuilder(t *testing.T) {
 		[]string{flag.Provider, flag.Region},
 	)
 }
+
+func TestCreateBuilderWithTier(t *testing.T) {
+	test.CmdValidator(
+		t,
+		CreateBuilder(),
+		0,
+		[]string{flag.Provider, flag.Region, flag.Tier},
+	)
+}
```

Comment:
update the original test should test all available flags

Reply:
Done in [292f8ad](https://github.com/mongodb/mongodb-atlas-cli/pull/2839/commits/292f8adf0500b4d189ad960b11c59c80ec717794) !

Reply:
not seeing the change on that commit

Reply:
So sorry about this, total mind blank. I've removed `TestCreateBuilderWithTier` and instead just have `TestCreateBuilder` with all arguments. Fixed the wrong test in a rush 😅 

[5df54a2](https://github.com/mongodb/mongodb-atlas-cli/pull/2839/commits/5df54a234371d95b95f992615eb123574a679697)